packages/charts/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.5.21](https://github.com/keen/keen/compare/@keen.io/charts@2.5.20...@keen.io/charts@2.5.21) (2021-01-06)
```

Comment:
Are You maybe invoked a script responsible for bumping a versions ? This should be done automatically via Lerna in CI pipeline.

Reply:
develop branch did not have the most recent changes in tag @keen.io/widgets@2.2.21 https://github.com/keen/keen/compare/develop...@keen.io/widgets@2.2.21

Reply:
Ok got it. The Storybook should works now - so we can right now test and check this PR.

Reply:
@maciejrybaniec this is now rebased to the latest version of develop branch. I started storybook and confirmed that there are no visual regressions in the widgets